core/commands/mount_darwin.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"syscall"
 
-	fuseversion "github.com/jbenet/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version"
+	fuseversion "github.com/jbenet/go-fuse-version"
```

Comment:
you somehow unvendored `go-fuse-version`. are you running straight up `make vendor`? 

Oh, you might not have the dep. it may be important to make it:

```
vendor: godep
	go get ./...
	godep save -r ./...
```

cc @briantigerchow